prereise/gather/demanddata/nrel_efs/tests/test_get_efs_data.py

```diff
@@ -1,107 +1,138 @@
-        df = pd.read_csv("EFSLoadProfile_Reference_Slow.csv")
-
-        # Access the columns
-        test_cols = list(df.columns)
-        exp_cols = [
-            "Electrification",
-            "TechnologyAdvancement",
-            "Year",
-            "LocalHourID",
-            "State",
-            "Sector",
-            "Subsector",
-            "LoadMW",
-        ]
-
-        # Compare the two values
-        assert len(test_cols) == len(exp_cols)
-
-        # Remove the downloaded EFS data set
-        os.remove("EFSLoadProfile_Reference_Slow.csv")
-
-    except FileNotFoundError:
-        # If the automated extraction did not work, check that the .zip file was created
-        assert os.path.isfile("EFSLoadProfile_Reference_Slow.zip")
-
-        # Remove the downloaded .zip file
-        os.remove("EFSLoadProfile_Reference_Slow.zip")
-
+    # Check that the expected .zip file was downloaded
+    assert os.path.isfile("project_resstock_efs_2013.zip")
 
-def test_download_flexibility_data():
-    # Download one of the EFS flexibility data sets
-    download_flexibility_data(es={"Reference"})
+    # Remove the downloaded .zip file
+    os.remove("project_resstock_efs_2013.zip")
 
-    try:
-        # Load the downloaded EFS flexibility data set
-        df = pd.read_csv("EFSFlexLoadProfiles_Reference.csv")
 
-        # Access the columns
-        test_cols = list(df.columns)
-        exp_cols = [
-            "Electrification",
-            "TechnologyAdvancement",
-            "Flexibility",
-            "Year",
-            "LocalHourID",
-            "State",
-            "Sector",
-            "LoadMW",
-        ]
+def test_extract_data():
+    # Try extracting the dummy .csv file from the dummy .zip file
+    _extract_data(
+        z=None,
+        zf_works=False,
+        zip_name="test.zip",
+        csv_name="test_demand.csv",
+        fpath=os.getcwd(),
+        sz_path="C:/Program Files/7-Zip/7z.exe",
+    )
 
-        # Compare the two values
-        assert len(test_cols) == len(exp_cols)
+    # Load the downloaded EFS demand data set
+    df = pd.read_csv("test_demand.csv")
 
-        # Remove the downloaded EFS data set
-        os.remove("EFSFlexLoadProfiles_Reference.csv")
+    # Access the columns
+    test_cols = list(df.columns)
+    exp_cols = [
+        "Electrification",
+        "TechnologyAdvancement",
+        "Year",
+        "LocalHourID",
+        "State",
+        "Sector",
+        "LoadMW",
+    ]
 
-    except FileNotFoundError:
-        # If the automated extraction did not work, check that the .zip file was created
-        assert os.path.isfile(
-            "EFS Flexible Load Profiles - Reference Electrification.zip"
-        )
+    # Compare the two values
+    assert len(test_cols) == len(exp_cols)
 
-        # Remove the downloaded .zip file
-        os.remove("EFS Flexible Load Profiles - Reference Electrification.zip")
+    # Remove the downloaded EFS data set
+    os.remove("test_demand.csv")
```

Comment:
Same comment as above: these should be part of a `finally` block.

Reply:
Done.